app/(app)/session/[session].tsx

```diff
@@ -1,40 +1,184 @@
 
+  // filter session by slug
+  const session = Sessions.data.filter((_session) => _session.slug === slug)[0];
+
+  const [showMoreBio, setShowMoreBio] = useState(false);
+
   return (
-    <MainContainer preset="scroll">
-      <Stack.Screen
-        options={{
-          title: `Session ${slug}`,
-          headerTitleAlign: 'center',
-          headerTintColor: colors.text,
-          headerStyle: {
-            backgroundColor: colors.background,
-          },
-          headerLeft: () => <AntDesign name="arrowleft" size={24} color={colors.text} onPress={() => router.back()} />,
-        }}
-      />
-
-      <View style={styles.main}>
-        <StyledText>session slug: {slug}</StyledText>
-      </View>
-    </MainContainer>
+    <View style={styles.main}>
+      <MainContainer preset="scroll">
+        <Stack.Screen
+          options={{
+            title: `Session Details`,
+            headerTitleAlign: 'center',
+            headerTintColor: colors.text,
+            headerStyle: {
+              backgroundColor: dark ? colors.bg : colors.background,
+            },
+            headerTitleStyle: {
+              fontSize: 18,
+              fontWeight: '300',
+            },
+            headerLeft: () => (
+              <AntDesign name="arrowleft" size={24} color={colors.text} onPress={() => router.back()} />
+            ),
+          }}
+        />
+
+        <View style={styles.main}>
+          <View style={[styles.centered, { borderColor: dark ? colors.background : colors.border }]}>
+            <Row>
+              <View>
+                <View style={styles.row}>
+                  <FontAwesome5 name="android" size={18} color={colors.tertiary} />
+                  <Space size={5} horizontal />
+                  <StyledText size="sm" font="regular" style={{ color: colors.tertiary }}>
+                    {session?.speakers && session?.speakers.length > 1 ? 'Speakers' : 'Speaker'}
+                  </StyledText>
+                </View>
+                <StyledText size="lg" font="bold" style={{ color: dark ? colors.text : colors.primary }}>
+                  {session?.speakers.map((speaker) => speaker.name).join(', ')}
+                </StyledText>
+              </View>
+              <View>
+                <Space size={14} />
+                <FontAwesome5 name="star" size={24} color={colors.primary} />
+              </View>
+            </Row>
+
+            <Space size={20} />
+
+            <StyledText size="lg" font="bold">
+              {session?.title}
+            </StyledText>
+
+            <Space size={16} />
+
+            {session?.speakers && session?.speakers.length > 1 ? (
+              session?.speakers.map((speaker, index) => (
+                <View key={index.toString()}>
+                  <StyledText font="regular" style={{ color: colors.primary }}>
+                    {speaker.name} - {''}
+                    <StyledText font="regular" style={{ color: colors.textLight }}>
```

Comment:
`textLight` has very low contrast with the background on dark mode and does not match the design. We can remove it to use the default text colour in `StyledText`.

<img width="573" alt="image" src="https://github.com/droidconKE/droidconKE2023ReactNative/assets/5578688/9ad51735-b883-44ed-9924-2947d3b0e09b">